oshc/templates/base.html

```diff
@@ -15,7 +15,7 @@
     <nav class="navbar navbar-default">
       <div class="container-fluid">
         <div class="navbar-header">
-           <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+           <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse">
```

Comment:
Wouldn't data-toggle and data-target achieve the functionality of collapsing links in the navbar in narrow viewports? How does collapsed class helps in this?